dev-tools/omdb/tests/successes.out

```diff
@@ -84,6 +84,24 @@ stderr:
 note: using database URL postgresql://root@[::1]:REDACTED_PORT/omicron?sslmode=disable
 note: database schema version matches expected (4.0.0)
 =============================================
+EXECUTING COMMAND: omdb ["db", "network"]
```

Comment:
It seems like this output would go in the "usage" test (if you want to cover it).  The point of the "successes" test is to sanity-check that the command produces reasonable-looking output against a real system.  You might add `omdb db network list-eips` to this test instead?  Or if it's too annoying to get that to work (you might have to add more redactions for the output to be stable across runs), skip it altogether?

Reply:
Moved this test to usage.